stable_baselines3/common/envs/bit_flipping_env.py

```diff
@@ -2,13 +2,13 @@
 from typing import Any, Dict, Optional, Union
 
 import numpy as np
-from gym import GoalEnv, spaces
+from gym import Env, spaces
 from gym.envs.registration import EnvSpec
 
 from stable_baselines3.common.type_aliases import GymStepReturn
 
 
-class BitFlippingEnv(GoalEnv):
+class BitFlippingEnv(Env):
```

Comment:
we should rather use `GoalEnv` from gym rotobics once https://github.com/Farama-Foundation/gym-robotics/issues/7 is solved, no?

Reply:
My intention was/is to remove the GoalEnv class from Gym in favor of the regular base class. We just kept it in gym-robotics because we didn't want to bother removing it. I would discourage other projects from using it.